imports/wasi_http/default_http/request.go

```diff
@@ -34,3 +48,32 @@ func (handler *Handler) handleFn(_ context.Context, mod api.Module, request, b,
 	}
 	return handler.res.MakeResponse(r)
 }
+
+// Handle handles HTTP client calls.
+// The remaining parameters (b..h) are for the HTTP Options, currently unimplemented.
+func (handler *Handler) handleFn_2023_10_18(_ context.Context, mod api.Module, request, b, c, d, e, f, g, h, ptr uint32) {
+	req, ok := handler.req.GetRequest(request)
+	if !ok {
+		msg := fmt.Sprintf("Failed to get request: %v\n", request)
+		log.Printf(msg)
+		doError(mod, ptr, msg)
+		return
```

Comment:
Is this condition expected to ever happen? Or would it be better suited to panic if we hit it?

Reply:
This condition will happen if someone loads a WASM binary that doesn't export the right function.

In such a situation, it feels like the right thing to do is have the server return a 500 error, as opposed to crashing via `panic()`